src/main/java/com/dumbdogdiner/stickyapi/bukkit/util/StartupUtil.java

```diff
@@ -52,7 +52,7 @@ public static boolean setupConfig(@NotNull JavaPlugin plugin) {
      * @param localeProvider The plugin's locale provider
      * @return False if something went wrong
      */
-    public static LocaleProvider setupLocale(@NotNull JavaPlugin plugin, @Nullable LocaleProvider localeProvider) {
+    public static @org.jetbrains.annotations.Nullable LocaleProvider setupLocale(@NotNull JavaPlugin plugin, @Nullable LocaleProvider localeProvider) {
```

Comment:
```suggestion
    public static @Nullable LocaleProvider setupLocale(@NotNull JavaPlugin plugin, @Nullable LocaleProvider localeProvider) {
```

Reply:
I'm surprised that this can be nullable. I would've thought it throws an exception if it fails - lowkey unrelated to this PR but worth noting.